src/main/index.ts

```diff
@@ -658,30 +652,23 @@ const router = t.router({
 
     return !isInitializedValidated;
   }),
-  holochainVersion: t.procedure.query(
-    () => HOLOCHAIN_MANAGERS[Object.keys(HOLOCHAIN_MANAGERS)[0]].version,
-  ),
-  isDevhubInstalled: t.procedure.query(() =>
-    Object.values(HOLOCHAIN_MANAGERS)
-      .flatMap((manager) => manager.installedApps)
-      .some((app) => app.installed_app_id === DEVHUB_APP_ID),
-  ),
+  holochainVersion: t.procedure.query(() => HOLOCHAIN_MANAGERS[DEFAULT_HOLOCHAIN_VERSION].version),
```

Comment:
maybe we should rename this to `defaultHolochainVersion` now.

Reply:
fixed